### PR TITLE
Add missing ErrorCount field to BareMetalHost CRD

### PR DESCRIPTION
**Pull request by andfasano**
Updating MAO BMH crd while CBO migration is still in progress. Related to https://github.com/openshift/cluster-baremetal-operator/pull/64.

### CODE REVIEW

**Comment by andfasano** [comment]
cc @zaneb @sadasu

File: install/0000_30_machine-api-operator_08_baremetalhost.crd.yaml [hunk 1]
```diff
@@ -705,6 +709,7 @@ spec:
                     type: string
                 type: object
             required:
+            - errorCount
```
Comment (zaneb) [inline install/0000_30_machine-api-operator_08_baremetalhost.crd.yaml:712]: Is this field being required going to be a problem for upgrades? i.e. does it mean that after updating the CRD, the baremetal-operator won't be able to write to the status until it is also upgraded?
If so this should probably be fixed upstream.

Comment (andfasano) [reply install/0000_30_machine-api-operator_08_baremetalhost.crd.yaml:713]: As discussed, I've added a default value for the field to allow smoother upgrade

**Comment by zaneb** [comment]
Upstream change is not accepted yet.
/hold

**Comment by andfasano** [comment]
Change merged upstream https://github.com/metal3-io/baremetal-operator/pull/722

/hold cancel

**Comment by zaneb** [comment]
/retitle Add missing ErrorCount field to BareMetalHost CRD
/retest

**Comment by andfasano** [comment]
Addressed by https://github.com/openshift/machine-api-operator/pull/767

**Pull request closed without merging**